hypervisor/Makefile

```diff
@@ -300,7 +296,7 @@ endif
 .PHONY: all
 all: lib $(HV_OBJDIR)/$(HV_FILE).32.out $(HV_OBJDIR)/$(HV_FILE).bin
 
-ifeq ($(CONFIG_PLATFORM_SBL),y)
+ifeq ($(FIRMWARE),sbl)
```

Comment:
I'm actually wondering if I understood all the changes correctly, do we need to use `FIRMWARE=` when calling `make` after these changes?

Reply:
For now `FIRMWARE` is only used in Makefile not code. Anyway I will totally remove these thing in the future. This is one part of my proposal.